### PR TITLE
Md Files

### PR DESCRIPTION
**Pull request by Shubhangi-cs**

### CODE REVIEW

File: docs/Ariba-batchsource.md [hunk 1]
```diff
@@ -23,8 +23,10 @@ annotating metadata, etc.
 **Realm (M)**: Realm name from which the data is to be extracted, for example, S4VALL.   
 **View Template Name (M)**: Name of the view template from which data is to be extracted, for example, RequestFactSystemView.
 
-## Credentials:
-
+## Connection:
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+**Connection:** Name of the connection to use. Template Names information will be provided by the connection.
```
Comment (robinrielley) [inline docs/Ariba-batchsource.md]: When I view the file, these properties do not appear on separate lines. Maybe there's a missing paragraph tag or something? Connection and Ariba Client ID (M) appear in the same paragraph as the Use Connection property.

Also, should it read Template Name information?  Based on the previous doc, seems like it is one Template per connection?

File: docs/Ariba-connector.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Ariba Connection
+
+Description
+-----------
+Use this connection to access data in Salesforce.
```
Comment (robinrielley) [inline docs/Ariba-connector.md]: Change Salesforce to Ariba?

Where's the Name property and Description property? The other connections let you enter a Name and Description for the connection?

File: docs/Ariba-connector.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Properties
+----------
+**Ariba Client ID (M)**: Ariba Client ID.
+
+**Ariba Client Secret (M)**: Ariba Client Secret.  
+
+**Ariba API Key (M)**: Ariba API Key.
+
+**API Endpoint (M)**: Base path of the Ariba API, for example, https://sandbox.api.sap.com.
```
Comment (robinrielley) [inline docs/Ariba-connector.md]: When I view this .md file, the URL has a link? Can the link be removed since it's not a real url?

**Review by robinrielley — CHANGES_REQUESTED** [review]
added another comment

**Review by robinrielley — CHANGES_REQUESTED** [review]
added new comment

File: docs/Ariba-connector.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+**Ariba API Key (M)**: Ariba API Key.
+
+**API Endpoint (M)**: Base path of the Ariba API, for example, https://sandbox.api.sap.com. 
+
+**System Type**: Type of system the Ariba instance is running on: Production or Sandbox.  
+
+**Realm (M)**: Realm name from which the data is to be extracted, for example, S4VALL.
+
+* **JSON**: Contents of the service account JSON file.
```
Comment (robinrielley) [inline docs/Ariba-connector.md]: I think you can delete JSON. Looks like a copy/paste error from a GCP plugin connection

**Review by robinrielley — CHANGES_REQUESTED** [review]
added new comment

File: docs/Ariba-batchsource.md (hunk 1 above)
Comment (Shubhangi-cs) [reply docs/Ariba-batchsource.md]: Done the changes

File: docs/Ariba-connector.md (hunk 2 above)
Comment (Shubhangi-cs) [reply docs/Ariba-connector.md]: Added

File: docs/Ariba-connector.md (hunk 3 above)
Comment (Shubhangi-cs) [reply docs/Ariba-connector.md]: removed

File: docs/Ariba-connector.md (hunk 4 above)
Comment (Shubhangi-cs) [reply docs/Ariba-connector.md]: Removed

File: docs/Ariba-batchsource.md [hunk 5]
```diff
@@ -23,8 +23,10 @@ annotating metadata, etc.
 **Realm (M)**: Realm name from which the data is to be extracted, for example, S4VALL.   
 **View Template Name (M)**: Name of the view template from which data is to be extracted, for example, RequestFactSystemView.
```
Comment (robinrielley) [inline docs/Ariba-batchsource.md:25]: This description has an example with a link, but it's not a real URL. Remove the link:
API Endpoint (M): Base path of the Ariba API, for example, https://sandbox.api.sap.com/.

**Review by robinrielley — APPROVED** [review]

**Pull request merged**